connections/sim/hw/xbee_module_sim.py

```diff
@@ -93,7 +92,8 @@ def send_to_rocket(self, data):
         rx_options = b"\x02"
         self.rocket_callback(
             self._create_frame(
-                FrameType.RX_INDICATOR, SOURCE_ADDRESS + reserved + rx_options + data,
+                FrameType.RX_INDICATOR,
```

Comment:
Wait, was this a bug?

Reply:
If so, im curious what changed that led to you finding it

Reply:
I believe previously I had the source address hard coded because it wasn't being used; when I removed that because its value now has importance, I found this reference to it